concepts/numbers/about.md

```diff
@@ -19,6 +19,32 @@ There is no separate integer type in common everyday use.
 The `bigint` type is not designed to replace the `number` type for everyday uses.
 `42` is still a `Number`, not a `BigInt`.
 
+## Special Notations
+
+There are two types of notations:
+- Exponential notation(E-notation)
+- Underscore notation
```

Comment:
This section can be removed, it is fine to just have the two sections below without introduction text.